packages/notification-center/src/components/novu-provider/NovuProvider.tsx

```diff
@@ -81,9 +86,9 @@ export function NovuProvider({
     (newSession: ISession) => {
       applyToken({ apiService, token: newSession.token });
       initializeSocket(newSession);
-      setSessionInitialized(true);
+      setSessionInfo((old) => ({ ...old, isSessionInitialized: true }));
```

Comment:
this is the callback called when the session is initialized... setting this flag to true will result in fetching the org, unseen count, and notifications...